nestjs-counter/apps/server/src/exchange-rates/exchange-rates.providers.ts

```diff
@@ -0,0 +1,37 @@
+import {
+  WorkflowClient,
+  WorkflowExecutionAlreadyStartedError,
+} from '@temporalio/client';
+import { taskQueue } from '@app/shared';
+
+export const exchangeRatesProviders = [
+  {
+    provide: 'WORKFLOW_CLIENT',
+    useValue: new WorkflowClient(),
```

Comment:
Let's use the new `Client` here instead.

Reply:
I'd show how to inject a configured `Connection`, not just a `Client` that uses localhost.